crates/dojo/world/Cargo.toml

```diff
@@ -24,6 +24,9 @@ dojo-types.workspace = true
 ipfs-api-backend-hyper = { git = "https://github.com/ferristseng/rust-ipfs-api", rev = "af2c17f7b19ef5b9898f458d97a90055c3605633", features = [ "with-hyper-rustls" ], optional = true }
 toml.workspace = true
 url.workspace = true
+hex.workspace = true
+hex-literal.workspace = true
+num-bigint.workspace = true
```

Comment:
those weren't removed from the `sozo/Cargo.toml`.

Reply:
Actually, could you uncomment and add to the `sozo/main.rs` this?
```rust
#![warn(unused_crate_dependencies)]
```
This will ensure we removed unused crates. 👍

Reply:
My bad, I forgot to check afterwards what I had taken out!

Reply:
Should I remove this for cleanliness? or keep it as a comment underneath? 
`#![cfg_attr(not(test))]`